src/worker/startup.ts

```diff
@@ -91,25 +90,24 @@ const setDelayOnRateLimiting = (jobHandler) => async (job: Job, logger: LoggerWi
 		throw err;
 	}
 };
-const sendQueueMetrics = async () => {
-	if (await booleanFlag(BooleanFlags.EXPOSE_QUEUE_METRICS, false)) {
```

Comment:
The [feature flag](https://hello.atlassian.net/browse/FD-26653) is currently enabled on all environments so we can just remove the if statement.